src/devTools.js

```diff
@@ -247,11 +250,14 @@ export const ActionCreators = {
   setMonitorState(monitorState) {
     return { type: ActionTypes.SET_MONITOR_STATE, monitorState };
   },
-  recomputeStates(committedState, stagedActions) {
+  recomputeStates(committedState, stagedActions, timestamps, currentStateIndex=-1, skippedActions={}) {
```

Comment:
Main use case was for import after export (check why it was introduced in the first place). At the moment if you export and then import you will loose timestamps - if you use the ActionCreator.recomputeStates and do not create your Action by "hand".

Also I think it might be nice to have a monitor which stores multiple "actionsequences" and provide a way to switch between them without using the clipboard - which is just export/import again using for example localstorage/sessionstorage...

Additionally you will loose the currentIndex and the skipped actions, this is not that important, as you can restore these by dispatching multiple toggle + one jumpto action, but if we open up the state anyway, why not provide these optionally.